daemon/cluster/convert/network.go

```diff
@@ -8,6 +8,7 @@ import (
 	types "github.com/docker/docker/api/types/swarm"
 	netconst "github.com/docker/libnetwork/datastore"
 	swarmapi "github.com/docker/swarmkit/api"
+	allocator "github.com/docker/swarmkit/manager/allocator"
```

Comment:
Makes sense. I am ok on duplicating the function in here as long as everybody agrees.